.github/workflows/ci.yml

```diff
@@ -383,9 +384,32 @@ jobs:
     # Includes hack to disable trunk tests on MacOS which are currently broken
     # Also, no detect_leaks support for ASAN
     - name: Run basic tests (MacOS)
-      if: ${{ runner.os == 'macOS' }}
+      if: ${{ matrix.env.TEST_TYPE == 'macos' }}
       run: |
         : > src/lib/server/trunk_tests.mk
         make test
       env:
         ASAN_OPTIONS: symbolize=1 detect_stack_use_after_return=1
+
+    # Fuzz each protocol for a short time, continuing on failures
+    # Aim to keep to overall runtime of this job in line with other jobs in the CI workflow
+    - name: Run fuzzer tests
+      if: ${{ matrix.env.TEST_TYPE == 'fuzzing' }}
+      run: |
+        # For fuzzing we won't be needing eapol_test
+        mkdir -p build/tests/eapol_test
+        : > build/tests/eapol_test/eapol_test.mk
+        CPUS=`nproc`
+        mkdir -p fuzzer_artifacts/radius && make -j $CPUS test.fuzzer.radius FUZZER_RUNTIME=240 FUZZER_ARTIFACTS=fuzzer_artifacts/radius/ FUZZER_JOBS=$CPUS FUZZER_WORKERS=$CPUS || : ; mv -f fuzz-*.log fuzzer_artifacts/radius/
```

Comment:
I think this belongs more in the build system.  This should really be calling a single target.

Reply:
@arr2036 Do you mean something like `make ci-fuzzer` to execute the fuzzing for each protocol?

Reply:
it already supports `make test.fuzzer`  which runs the fuzzer tests for all protocols, for 10 seconds.  You can just do `make -j 4 test.fuzzer`, and it should run all 4 jobs in parallel.

The build system should put the outputs into `artifacts/$(PROTOCOL)/`

I'm also not sure that setting `FUZZER_RUNTIME=240` here will do anything.  The `src/bin/all.mk` file has `FUZZER_RUNTIME := 0`, which _always_ sets it to zero, no matter what the environment variables say.